cmd/run.go

```diff
@@ -90,12 +101,16 @@ func (h *Handler) Run(ctx context.Context, req *entity.CommandRequest) error {
 
 	hasDockerfile := true
 
+	if _, err := os.Stat(fmt.Sprintf("%s/%s", pwd, dockerfile)); os.IsNotExist(err) {
+		dockerfile = "Dockerfile"
+	}
+
 	if _, err := os.Stat(fmt.Sprintf("%s/Dockerfile", pwd)); os.IsNotExist(err) {
```

Comment:
I am seeing errors when running in project that only has an alternativley name Dockerfile. I think it is because it is checking here if `./Dockerfile` exists. We probably want to check `./${dockerfile}` instead.

![image](https://user-images.githubusercontent.com/3044853/168182769-8d107b38-2851-4f1d-92d8-49b1f74235e0.png)